src/components/Navigation/Navigation.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<nav class="container mx-auto py-6">
+	<nav class="container flex px-6 md:flex-none justify-start mx-auto py-6">
```

Comment:
Isn't the aligning to left default behaviour?

Reply:
Alright, just remember to set it back to 0 on larger screens. Always do these tasks so nothing changes on the large devices.